src/components/Book.css

```diff
@@ -0,0 +1,94 @@
+.list-container {
```

Comment:
- [ ] Please as in Zeplin's design the width of the add book form section and the books cards section should be the same, please adjust the width of add book section and the width of the input fields to match the design.
You design
![image](https://user-images.githubusercontent.com/74744787/222929192-e29042dd-65e9-4c77-b83c-14aa04ae508a.png)
![image](https://user-images.githubusercontent.com/74744787/222929211-7ea78d6f-1447-498e-8225-3f34e88000ba.png)